app/controllers/api/subcollections/tags.rb

```diff
@@ -85,7 +85,7 @@ def parse_tag(data)
 
       def parse_tag_from_href(data)
         href = data["href"]
-        tag  = if href && href.match(%r{^.*/tags/#{CID_OR_ID_MATCHER}$})
+        tag  = if href && href.match(%r{^.*/tags/#{ArRegion::CID_OR_ID_MATCHER}$})
```

Comment:
Looks like there's a few of these around....perhaps this could be addressed in separate PR so it could be backported if needed? I wonder if it regressed when this became a mixin?